pkgs/games/kobodeluxe/default.nix

```diff
@@ -9,14 +9,31 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ SDL SDL_image libGLU libGL ];
+  nativeBuildInputs = [ makeWrapper ];
 
   prePatch = ''
     sed -e 's/char \*tok/const char \*tok/' -i graphics/window.cpp
   '';
 
-  patches = [ ./glibc29.patch ];
+  postInstall = ''
+    wrapProgram $out/bin/${meta.mainProgram} \
+      --run "mkdir -p ~/.config/${pname}" \
+      --add-flags "-scores ~/.config/${pname}"
+  '';
+
+  patches = [
+    (fetchpatch {
+      url = "https://sources.debian.org/data/main/k/${pname}/${version}-10/debian/patches/04_enemies-pipe-decl.patch";
+      sha256 = "sha256-myMhB7+7y6EeD+xIfeHTbJPLrulyQmzpTxuNX1azpSY=";
+    })
+    (fetchpatch {
+      url = "https://sources.debian.org/data/main/k/${pname}/${version}-10/debian/patches/ignore-appinputfocus.patch";
+      sha256 = "sha256-RTLeF2hvQJ4F9+dJvTL25VXD1RFrJsmcDsZSIYOhYYo=";
+    })
+  ];
```

Comment:
These need comments why they're necessary.

Reply:
The patches explain why they're necessary. The description of the PR also does.

> Fixes an issue with the game being stuck in the pause state: [this](https://sources.debian.org/patches/kobodeluxe/0.5.1-10/ignore-appinputfocus.patch/) patch from Debian fixes that.

The other patch was already there and it doesn't build without it

Again, I'm not sure whether you're asking for comments in the source or comments here.

Reply:
Always comments in code.

PR/commit comments are nice for communicating why something was done in a certain way but those are not accessible to people reading the code.

When this package gets refactored by somone later, they shouldn't need to dig up this PR to know wheher the patches are still necessary.

Reply:
Sure, no problem, I'll add a comment. Although as I said the patches themselves describe the changes. If you download the patch there's a description explaining it.